src/core/thread/osthread.d

```diff
@@ -820,6 +820,7 @@ class Thread
                     if ( pthread_create( &m_addr, &attr, &thread_entryPoint, cast(void*) this ) != 0 )
                         onThreadError( "Error creating thread" );
                 }
+                pthread_attr_destroy( &attr ) && assert(false);
```

Comment:
The remaining error handling uses `onThreadError`. Why use a plain `assert` (without a message) here?

Reply:
There's no particular reason and it should be. Thanks for pointing out!